kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8/kafkaproxyingresses.kroxylicious.io-v1.yml

```diff
@@ -79,8 +79,8 @@ spec:
                 loadBalancer:
                   type: object
                   description: |-
-                    clusterIP specifies that this ingress is for access within the same Kubernetes cluster using 
-                    ClusterIP Kubernetes Services.
+                    loadBalancer specifies that this ingress is for access from outside the Kubernetes cluster
```

Comment:
the service can also be used internally, but figured it was a bit noisy to talk about that here when the purpose is really enabling off-cluster. WDYT?